python/ray/serve/api.py

```diff
@@ -1044,6 +1018,11 @@ def prev_version(self) -> Optional[str]:
         """
         return self._prev_version
 
+    @property
+    def config(self) -> DeploymentConfig:
+        """Configs used for the deployment."""
+        return self._config
```

Comment:
I don't think this contains all of the things that can be passed to `.options`.

Also, let's not make it a public API please